### DISCUSSION TITLE
Have mpv auto start streams?

### DISCUSSION THREAD

**Discussion by DannyDarth** [streamlink/streamlink-twitch-gui#973] [Q&A]
Is there a custom parameter that will have streams auto start using mpv? Everytime i load up a stream i have to click the little chat bubble at the bottom twice to get it to start playing.

Right now i have parameters for full screen on a monitor:
mpv --fs --fs-screen=2

I want it to work with this.

**Comment by bastimeyer** [comment]
You must've set something unintentional in your mpv config file. mpv should start playing automatically in its default config.

https://mpv.io/manual/stable/

**Reply by DannyDarth** [reply]
So im looking in the programdata folder of chocolatey, which is the only play im finding mpv. I do not see an mpv config. Should i just make it?

**Reply by bastimeyer** [reply]
I don't use Windows. You'll have to read the manual I've just linked. Good luck,

https://mpv.io/manual/stable/#files-on-windows
